injector/network_disruption.go

```diff
@@ -70,19 +104,25 @@ func NewNetworkDisruptionInjector(spec v1beta1.NetworkDisruptionSpec, config Net
 		config.DNSClient = network.NewDNSClient()
 	}
 
-	return networkDisruptionInjector{
+	config.State = DisruptionState{}
```

Comment:
Could we set `config.State.State` to created here? (or does that cause problems)